# IP address of the hardware
string ip_address

# Indexing mode
uint8 indexing_mode

# Motion scale factor (default 1.0) 
float32 speed_factor

# Force scale factor (default 1.0)
float32 force_factor

# Max force (in Newtons)
float32 max_force

# Max torque (in Newton-meters)
float32 max_torque

# Enable force-feedback (for debug purposes, default True)
bool power_enable

# Coupling gains in translation and rotation
# Ktrans in Newton/meter
# BTrans in Newton/(meter/second)
# KRot in Newton.meter/radian
# BRot in Newton.meter/(radian/second)
float32 Ktrans
float32 Btrans
float32 Krot
float32 Brot

# Transformation of the base of the Virtuose
geometry_msgs/Transform base_frame

---

uint32 client_id
bool success
bool error


